utility/src/main/java/org/oppia/android/util/parser/CustomBulletSpan.kt

```diff
@@ -21,10 +21,15 @@ class CustomBulletSpan(context: Context) : LeadingMarginSpan {
   private val bulletRadius: Int
   private val gapWidth: Int
   private val yOffset: Int
+  /** The space between the parent and the bullet. */
   private val spacingBeforeBullet: Int
+  /** The space between the bullet and the text. */
   private val spacingBeforeText: Int
+  /** The total spacing between the bullet and the text.
+   *  totalSpacingToTextStart - includes spacingBeforeBullet, spacingBeforeText and double of bulletRadius
```

Comment:
This bit seems unnecessary--suggest removing it (the reader can always look at how it's computed to figure out the specifics).

Reply:
Done